wbooks-api/spec/services/open_library_service_spec.rb

```diff
@@ -0,0 +1,52 @@
+require 'rails_helper'
+require 'support/mocks_and_stubs/open_library_request'
```

Comment:
I think that if you add `Dir[Rails.root.join('spec/support/**/*.rb')].each { |f| require f }
` this won't be needed in every file